shell/platform/android/io/flutter/app/FlutterActivity.java

```diff
@@ -151,4 +151,10 @@ public boolean loadIntent(Intent intent) {
     public FlutterView getFlutterView() {
       return flutterView;
     }
+
+    @Override
+    public void onTrimMemory(int level) {
+        if (level == TRIM_MEMORY_RUNNING_LOW)
```

Comment:
Something like would be great:
A trim level delivered only when the application is running must be selected as that gives the framework a change to react to the notification.